src/emulator/modules/SceLibKernel/src/SceLibKernel.cpp

```diff
@@ -319,12 +319,12 @@ EXPORT(int, sceIoIoctl) {
     return unimplemented("sceIoIoctl");
 }
 
-EXPORT(int, sceIoIoctlAsync) {
-    return unimplemented("sceIoIoctlAsync");
+EXPORT(int, sceIoLseek, SceUID fd, SceOff offset, int whence) {
+    return seek_file(fd, offset, whence, host.io);
 }
 
-EXPORT(int, sceIoLseek) {
-    return unimplemented("sceIoLseek");
+EXPORT(int, sceIoIoctlAsync) {
```

Comment:
The function ordering got swapped around, so regenerating modules will cause merge problems.